backend/src/test/kotlin/app/ehrenamtskarte/backend/application/webservice/Verein360ApplicationTest.kt

```diff
@@ -0,0 +1,208 @@
+package app.ehrenamtskarte.backend.application.webservice
+
+import app.ehrenamtskarte.backend.GraphqlApiTest
+import app.ehrenamtskarte.backend.application.database.ApplicationEntity
+import app.ehrenamtskarte.backend.application.database.ApplicationVerificationEntity
+import app.ehrenamtskarte.backend.application.database.ApplicationVerificationExternalSource
+import app.ehrenamtskarte.backend.application.database.ApplicationVerifications
+import app.ehrenamtskarte.backend.application.database.Applications
+import app.ehrenamtskarte.backend.application.webservice.schema.create.Application
+import app.ehrenamtskarte.backend.application.webservice.schema.create.ApplicationType
+import app.ehrenamtskarte.backend.auth.database.ApiTokenType
+import app.ehrenamtskarte.backend.auth.database.ApiTokens
+import app.ehrenamtskarte.backend.helper.TestAdministrators
+import app.ehrenamtskarte.backend.helper.TestApplicationBuilder
+import app.ehrenamtskarte.backend.helper.TestData
+import app.ehrenamtskarte.backend.util.GraphQLRequestSerializer
+import io.javalin.testtools.JavalinTest
+import org.jetbrains.exposed.sql.deleteAll
+import org.jetbrains.exposed.sql.selectAll
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.MethodSource
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+
+internal class Verein360ApplicationTest : GraphqlApiTest() {
+
+    data class ValidationErrorTestCase(val application: Application, val error: String)
+
+    companion object {
+        @JvmStatic
+        fun validationErrorTestCases(): List<ValidationErrorTestCase> {
```

Comment:
i would add here the policy opt in
`givenInformationIsCorrectAndComplete` to check that its set

Reply:
I will open a separate PR for that, it requires some additional work

Reply:
The problem with these fields is that they are validated in init blocks, which means, it's not possible to create an application object with invalid values, such as givenInformationIsCorrectAndComplete = false.
I think I would have to re-work the current approach and build an application as a string rather than as an object.
Or create a default object, serialize it to a string, and then replace the values in the string with invalid values.
Not sure yet what is less ugly.
I will create another task for that.